cli/cmd/help.go

```diff
@@ -114,10 +115,11 @@ func getExternalCommandsString(modulesInfo *modules.ModulesInfo) string {
 	}
 
 	if str != "" {
+		str = util.Bold("\nEXTERNAL COMMANDS\n") + str
 		return str[:len(str)-1]
```

Comment:
What is the purpose of `:len(str)-1`? Can we use Trim instead?

Reply:
Replaced